pkg/skaffold/runner/v2/dev.go

```diff
@@ -269,6 +276,7 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 			return fmt.Errorf("watching test files: %w", err)
 		}
 	}
+	// TODO(yuwenma): watch for render resources.
```

Comment:
```suggestion
	// TODO(yuwenma)[07/23/21]: watch for render resources.
```

Reply:
ah, this todo is also resolved in the next PR diff-based from this one.